producers/zap_producer/main.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+
+	v1 "github.com/thought-machine/dracon/api/proto/v1"
+	"github.com/thought-machine/dracon/producers/zap_producer/types"
+
+	"fmt"
+	"log"
+
+	"github.com/thought-machine/dracon/producers"
+
+)
+
+func main() {
+	if err := producers.ParseFlags(); err != nil {
+		log.Fatal(err)
+	}
+
+	inFile, err := producers.ReadInFile()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var results types.ZapOut
+	if err := producers.ParseJSON(inFile, &results); err != nil {
+		log.Fatal(err)
+	}
+
+	if err := producers.WriteDraconOut(
+		"zap",
+		parseOut(&results),
+	); err != nil {
+		log.Fatal(err)
+	}
```

Comment:
```suggestion
	if err := producers.WriteDraconOut("zap", parseOut(&results)); err != nil {
		log.Fatal(err)
	}
```

Might as well keep this on a single line for consistency with the lines above, since it's not too long.